docs/user_guide/metrics.md

```diff
@@ -193,8 +193,8 @@ If building Triton locally, the `TRITON_ENABLE_METRICS_GPU` CMake build flag can
 
 |Category      |Metric          |Metric Name |Description                            |Granularity|Frequency    |
 |--------------|----------------|------------|---------------------------|-----------|-------------|
-|GPU Utilization |Power Usage   |`nv_gpu_power_usage` |GPU instantaneous power                |Per GPU    |Per interval   |
-|              |Power Limit     |`nv_gpu_power_limit` |Maximum GPU power limit                |Per GPU    |Per interval   |
+|GPU Utilization |Power Usage   |`nv_gpu_power_usage` |GPU instantaneous power, in Watts      |Per GPU    |Per interval   |
+|              |Power Limit     |`nv_gpu_power_limit` |Maximum GPU power limit, in Watts      |Per GPU    |Per interval   |
 |              |Energy Consumption|`nv_energy_consumption` |GPU energy consumption in joules since Triton started|Per GPU|Per interval|
```

Comment:
Also, if you have the chance, would you mind adding a comma here before "in joules"? I don't care one way or the other about the comma, but would like to be consistent within the table/document.

Note: not required for approval, as this is outside the scope of your change.

Reply:
Added comma, fixed capitalization, re-aligned GPU metric table. I know not all tables are aligned, but some have quite long descriptions that it's somewhat moot for those anyway.
![image](https://github.com/triton-inference-server/server/assets/21284872/d66d5061-8288-488d-8f48-8ad25baa79b0)